test/account.c

```diff
@@ -99,8 +99,8 @@ int test_account_uri_complete(void)
 		{ "user@domain.com:5677",
 		  "sip:user@domain.com:5677" },
 
-		{ "sip:user",
-		  "sip:user@proxy.com" },
+		{ "sip:office.local",
+		  "sip:office.local" },
```

Comment:
please keep the old testcase, and add office.local as an extra case

Reply:
The original test case has the problem that it will fail. We decided together to change the behavior.

The URI "sip:user" is a bad example, because `user` would be in this case a domain, not a user.